setup.py

```diff
@@ -1,19 +1,15 @@
+from pathlib import Path
 from setuptools import find_packages
 from setuptools import setup
 
 
-with open("README.rst") as myfile:
-    readme = myfile.read()
-with open("CHANGES.rst") as myfile:
-    changes = myfile.read()
-long_description = readme + "\n" + changes
-
-
 setup(
     name="Products.isurlinportal",
     version="3.0.1.dev0",
     description="Implementation of isURLInPortal method in Plone",
-    long_description=long_description,
+    long_description=(
+        f"{Path("README.rst").read_text()}\n{Path("CHANGES.rst").read_text()}\n"
```

Comment:
These are double quotes within double quotes, so this will fail.  Interestingly and surprisingly, the GHA jobs on Python 3.13 pass.  Locally 3.12 passes too.  Python 3.11 and lower fail.  On [GHA](https://github.com/plone/Products.isurlinportal/actions/runs/17645262760/job/50141569019?pr=40):

```
  File "/home/runner/work/Products.isurlinportal/Products.isurlinportal/.tox/.pkg/lib/python3.10/site-packages/setuptools/build_meta.py", line 317, in run_setup
    exec(code, locals())
  File "<string>", line 11
    f"{Path("README.rst").read_text()}\n{Path("CHANGES.rst").read_text()}\n"
             ^^^^^^
SyntaxError: f-string: unmatched '('
```

```suggestion
        f"{Path('README.rst').read_text()}\n{Path('CHANGES.rst').read_text()}\n"
```